app/views/lti_nrps/sync_roster.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Sync Course Roster</h2>
```

Comment:
This feels like significant duplication with `upload_roster.html.erb`.

I think it would be worthwhile to define a new partial to be shared across both pages (see: https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables)

Reply:
I agree, created the `courses/_updateRosterPreview.html.erb` partial

Reply:
Verified that roster displays correctly for both importing and syncing